### PR TITLE
Add units for the mass density derivative with respect to enthalpy 

### PR DESCRIPTION
**Pull request by caio-ramos**
DCC-93

This is an intermediate step toward the conclusion of DCC-93. Flash exported from PVTLib now has new outputs regarding derivatives with respect to enthalpy. This PR aims to cover the units of these derivatives including a new category and quantity type.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,8 +1,14 @@
+1.19.0 (2023-08-10)
+-------------------
+
+* Define the category and quantity type ``density derivative in respect to enthalpy``.
+* Add the default unit ``kg2/m3.J``.
```
Comment (nicoddemus) [inline CHANGELOG.rst:5]:
```suggestion
* Define the category and quantity type ``density derivative in respect to enthalpy`` (default unit: ``kg2/m3.J``).
```

**Comment by nicoddemus** [comment]
Ahh I see you plan to make the release from this branch (given the branch name), correct?

That's fine, just make sure to merge (instead of squash) to preserve the tag on `master`.

File: CHANGELOG.rst (hunk 1 above)
Comment (caio-ramos) [reply CHANGELOG.rst]: Thanks, done.

**Comment by caio-ramos** [comment]
> Ahh I see you plan to make the release from this branch (given the branch name), correct?
> 
> That's fine, just make sure to merge (instead of squash) to preserve the tag on `master`.

Exactly, thanks for the tip.

**Review by nicoddemus — APPROVED** [review]
Make sure to squash your commits into this PR to cleanup history (before tagging/releasing of course).

**Review by eniohayashi — APPROVED** [review]

**Pull request merged**